docs/reference/filebeat/filebeat-input-streaming.md

```diff
@@ -207,6 +207,20 @@ filebeat.inputs:
   url: wss://localhost:443/_stream
 ```
 
+## Keep Alive Configuration
+
+The `streaming` input currently supports keep-alive configuration options for streams of `type: websocket`. Using these configuration options we can further optimize the stability
+of our websocket connections and prevent them from idling out. 
+
+The `keep_alive` setting has the following configuration options:
+
+* `enable`: This indicates whether keep-alive is enabled. By default, this is set to `false`.
+* `interval`: This is the interval between keep-alive messages. This is a time duration value and by default, this is set to `30s`.
+* `write_control_deadline`: This is the deadline for writing control frames (like `PING`, `PONG`, or `CLOSE`) on a websocket connection. This timeout helps prevent indefinite blocking when the server or client is not responding to control frame requests. This is a time duration value and by default, this is set to `10s`.
```

Comment:
```suggestion
* `enable`: Indicates whether keep-alive is enabled. By default, this is set to `false`.
* `interval`: Interval between keep-alive messages, expressed as a time duration value. The default value is `30s`.
* `write_control_deadline`: Deadline for writing control frames, like `PING`, `PONG`, or `CLOSE`, on a websocket connection. The timeout, expressed as a time duration value, helps prevent indefinite blocking when the server or client is not responding to control frame requests. The default value is `10s`.
```

Reply:
`WebSocket` and `Keep-Alive` must be capitalized because they're proper nouns for their respective techs.